### PR TITLE
Pass suggestionIndex to onSuggestionSelected

### PR DESCRIPTION
**Pull request by oyeanuj**
Hi @moroshko! 

I’ve needed the `suggestionIndex` of the selected suggestion a couple of times in my application. 

So, I figured I would make a quick PR to see if this is something you’d like to add to the library.

Thank you for providing this library!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -351,6 +351,7 @@ where:
 
 * `suggestion` - the selected suggestion
 * `suggestionValue` - the value of the selected suggestion (equivalent to `getSuggestionValue(suggestion)`)
+* `suggestionIndex` - the index of the selected suggestion in the `suggestions` array )
```
Comment (moroshko) [inline README.md]: The `)` at the end should probably not be there. Also, `suggestionIndex` need to be added to the signature few lines above.

Comment (oyeanuj) [inline README.md]: Fixed in the latest commit.

**Comment by oyeanuj** [comment]
So, I had two different use-cases - 

1. I needed to get the original search result from the redux store, and having the key was a little more convenient than finding by value.

2. To get some usability insights into how people are finding and using the results, it is useful to log how relevant are the search results to the term the users entered.

I figured other people might find other uses as well, so worth putting it out there.

Let me know what you think!

File: README.md [hunk 2]
```diff
@@ -344,13 +344,14 @@ where:
 This function is called when suggestion is selected. It has the following signature:
 
 ```js
-function onSuggestionSelected(event, { suggestion, suggestionValue, sectionIndex, method })
+function onSuggestionSelected(event, { suggestion, suggestionValue, suggestionIndex,sectionIndex, method })
```
Comment (moroshko) [inline README.md]: Please add a space before `sectionIndex` to keep a consistent style.

Comment (oyeanuj) [inline README.md]: My bad, should be fixed now.

**Comment by moroshko** [comment]
It's in [v7.1.0](https://github.com/moroshko/react-autosuggest/releases/tag/v7.1.0) now.

**Pull request merged**